test/functional/testlib.bash

```diff
@@ -883,11 +883,6 @@ create_bundle() {
 	# set default values
 	bundle_name=${bundle_name:-$(generate_random_name test-bundle-)}
 	version=${version:-10}
-	if [ -z "$dir_list" ] && [ -z "$file_list" ] && [ -z "$link_list" ] && [ -z "$dangling_link_list" ] ; then
```

Comment:
In the commit message: s/unecessary/unnecessary/

Reply:
@otaviobp I fixed the typo in the commit message